File: assertion-roulette/CSharpXUnit/CSharpXUnit.Tests/Tests/Can_Write_Verbose_Message_With_Default_Verbosity.cs
using Xunit;

namespace DefaultNamespace
{
    public class LogExtensionsTests
    {
        [Fact]
        public void Can_Write_Verbose_Message_With_Default_Verbosity()
        {
            {
                // Given
                var log = new TestLog();
                // When
                log.Verbose("Hello World");
                // Then
                Assert.Equal(Verbosity.Verbose, log.Verbosity);
                Assert.Equal(LogLevel.Verbose, log.Level);
                Assert.Equal("Hello World", log.Message);
            }
        }
    }
}

File: refactored/assertion-roulette/java-junit/Can_Write_Verbose_Message_With_Default_Verbosity.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class LogExtensionsTests {

    @Test
    public void can_Write_Verbose_Message_With_Default_Verbosity() {
        {
            // Given
            TestLog log = new TestLog();
            // When
            log.verbose("Hello World");
            // Then
            assertEquals("Explanation message", Verbosity.Verbose, log.getVerbosity());
            assertEquals("Explanation message", LogLevel.Verbose, log.getLevel());
            assertEquals("Explanation message", "Hello World", log.getMessage());
        }
    }
}
